### PR TITLE
r?fzzzy Add a query to the /admin screen that shows retention

### PR DESCRIPTION
**Pull request by ianb**
Of course the auth on admin isn't working (to test I commented out the auth), and it probably doesn't render that well (locally I didn't have very interesting records to summarize), but it's a start.

### CODE REVIEW

File: server/src/pages/admin/view.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ class Head extends React.Component {
 
 class Body extends React.Component {
   render() {
+    console.log("all props", this.props);
```
Comment (fzzzy) [inline server/src/pages/admin/view.js]: Whoops, should remove this.

File: server/src/pages/admin/view.js [hunk 2]
```diff
@@ -66,6 +72,106 @@ class Body extends React.Component {
...
+        {buckets}
+      </div>
+    );
+  }
+}
+
+class RetentionByDate extends React.Component {
+  render() {
+    let months = ["", "Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
+    console.log("date props", this.props);
```
Comment (fzzzy) [inline server/src/pages/admin/view.js]: This one too...

**Comment by fzzzy** [comment]
r+ with console.log lines removed.

**Comment by ianb** [comment]
Also removed some dead code that I didn't manage to debug in the previous commit (date formatting code)

**Pull request merged**